pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/regionalcomputeforwardingrule/_generated_object_regionalcomputeforwardingrule.golden.yaml

```diff
@@ -34,9 +19,7 @@ spec:
     addressRef:
       name: computeraddress-${uniqueId}
   ipProtocol: ESP
-  loadBalancingScheme: EXTERNAL
   location: us-central1
-  resourceID: computeregionalforwardingrule-${uniqueId}
```

Comment:
Regarding resoruceID, here's the change (internal) https://docs.google.com/document/d/1A6YSEygeVJuXefb9cH52RiqEqp15nI5-UHNqG1T08kg/edit?tab=t.0#heading=h.yro65jsl04g3. You can focus on adding the `status.externalRef` for now.